index.toml

```diff
@@ -286,3 +286,14 @@ aliases = []
 completion_time = "15 min"
 created_at = 2023-11-30
 haystack_2 = true
+
+[[tutorial]]
+title = "Serialize Your Pipelines"
```

Comment:
Let's make it the same with the nb title. We can maybe say "Haystack" instead of "LLM". Up to you
```suggestion
title = "Serializing LLM Pipelines"
```